control/bgp.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.3
+info:
+  title: BGP Control models
+  description: >-
+    All BGP control schemas
+  version: ^0.0.0
+components:
+  schemas:
+    BGP.Notification:
+      description: >-
+        A NOTIFICATION message is sent when an error is detected with the BGP session, such as hold timer expiring, misconfigured AS number 
+        or a BGP session reset is requested. This causes the BGP connection to close. Send user defined NOTIFICATIONs for list of specified 
+        BGP peers. A user can send IANA defined BGP NOTIFICATIONs according to https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml. 
+        If a user wants to send custom Error Code and Error Subcode the custom object should be used.
+      type: object
+      properties:
+        names:
+          description: >-
+            The names of BGP Peers to send NOTIFICATION to. If no name is specified then NOTIFICATION will be send to all BGP peers that matches with the
```

Comment:
"If no name is specified then NOTIFICATION will be send to all BGP peers that matches with the x-constraint." -> "If no name is specified then NOTIFICATION will be sent to all configured BGP peers."

Reply:
updated